tools/astarte_export/config/config.exs

```diff
@@ -1,32 +1,7 @@
-# This file is responsible for configuring your application
-# and its dependencies with the aid of the Mix.Config module.
-use Mix.Config
+import Config
 
-# This configuration is loaded before any dependency and is restricted
-# to this project. If another project depends on this project, this
-# file won't be loaded nor affect the parent project. For this reason,
-# if you want to provide default values for your application for
-# third-party users, it should be done in your "mix.exs" file.
-
-# You can configure your application as:
-#
-#     config :astarte_export, key: :value
-#
-# and access this configuration in your application as:
-#
-#     Application.get_env(:astarte_export, :key)
-#
-# You can also configure a third-party app:
-#
-#     config :logger, level: :info
-#
-
-# It is also possible to import configuration files, relative to this
-# directory. For example, you can emulate configuration per environment
-# by uncommenting the line below and defining dev.exs, test.exs and such.
-# Configuration from the imported file will override the ones defined
-# here (which is why it is important to import them last).
-#
+config :cqerl,
+  cassandra_nodes: [{System.get_env("CASSANDRA_DB_HOST"), System.get_env("CASSANDRA_DB_PORT")}]
```

Comment:
Xandra, too, should receive the info on nodes and port (see https://hexdocs.pm/xandra/0.13.1/Xandra.html#start_link/1-options and https://github.com/astarte-platform/astarte_data_access/blob/45d4d20ae662751f47f4b8f2f9d5e302d5485ea9/lib/astarte_data_access/config.ex#L158-L168)